### PR TITLE
Removed redirects to same destination

### LINKED ISSUE #3521: URLs have several sources (duplicates in redirects)
# Bug report

## Description

The `jekyll doctor` tool reports about [429 conflicts](https://gist.github.com/dshevtsov/9f9a6eaef2cf454b5989ef8f3dcfee66) where a URL is destination for several sources. In most cases the conflicts are caused by duplicated redirects when the same URL is declared as a redirect in several topics. Some of them are introduced manually and can be fixed manually. Many of the conflicts can be caused by symlinks that requires a more complex system solution, then manual editing.

Primary goal: Find all files that do not have sym link AND have multiple redirects and ensure that they only have a single redirect.

- need to have 1 md file as the redirect per old file link
- for symlinks, add redirect for all versions of file being redirected
- may require multiple PRs to handle sections of links, see REPORT link below
- if you enter PR for section of links, make sure to note which ones you fix in the PR/Ticket

### Steps to reproduce

Run `bundle exec jekyll doctor`

## Expected result

Good: A search for conflicted URL returns only one source for the URL.
Perfect: No URLs with conflicts.

## Possible solutions

Search for a URL with conflicts and remove extra declarations so that the URL had one source only.

## Additional information

[Report with conflicts](https://gist.github.com/dshevtsov/9f9a6eaef2cf454b5989ef8f3dcfee66).

### PR DESCRIPTION
**Pull request by dshevtsov**
## Purpose of this pull request

This pull request removes extra redirects where possible. This fixes #3521

### CODE REVIEW

File: guides/v2.2/cloud/cdn/configure-fastly.md [hunk 1]
```diff
@@ -142,7 +140,7 @@ You cannot use the Fastly service in Integration environments.
 3.	Enter the following command to fully update and clear the cache:
 
 		php bin/magento setup:upgrade && php bin/magento cache:clean
-4. Edit your composer.json and ensure the Fastly module is included with version.
+1. Edit your composer.json and ensure the Fastly module is included with version.
```
Comment (dshevtsov) [inline guides/v2.2/cloud/cdn/configure-fastly.md]:
```suggestion
4. Edit your composer.json and ensure the Fastly module is included with version.
```

File: guides/v2.2/install-gde/composer.md [hunk 2]
```diff
@@ -32,8 +28,8 @@ Before you continue, you must do the following:
 To get the Magento metapackage:
 
 1.  Log in to your Magento server as, or switch to, the [Magento file system owner]({{page.baseurl}}/install-gde/prereq/file-sys-perms-over.html).
-1.  Change to the web server docroot directory or a directory that you have configured as a virtual host docroot.
-1.  Create a new Composer project using the {{site.data.var.ce}} or {{site.data.var.ee}} metapackage.
+2.  Change to the web server docroot directory or a directory that you have configured as a virtual host docroot.
```
Comment (dshevtsov) [inline guides/v2.2/install-gde/composer.md]:
```suggestion
1.  Change to the web server docroot directory or a directory that you have configured as a virtual host docroot.
```

File: guides/v2.2/install-gde/composer.md [hunk 3]
```diff
@@ -32,8 +28,8 @@ Before you continue, you must do the following:
...
+3.  Create a new Composer project using the {{site.data.var.ce}} or {{site.data.var.ee}} metapackage.
```
Comment (dshevtsov) [inline guides/v2.2/install-gde/composer.md]:
```suggestion
1.  Create a new Composer project using the {{site.data.var.ce}} or {{site.data.var.ee}} metapackage.
```

File: guides/v2.2/cloud/cdn/cloud-fastly-custom-response.md [hunk 4]
```diff
@@ -63,9 +60,9 @@ Verify that the modified source displays correctly in the browser. Then, add the
 
 1.  [Log in to the Magento Admin UI]({{ page.baseurl }}/cloud/onboarding/onboarding-tasks.html#admin).
 
-1.  Click **Stores** > **Settings** > **Configuration** > **Advanced** > **System**.
+2.  Click **Stores** > **Settings** > **Configuration** > **Advanced** > **System**.
```
Comment (dshevtsov) [inline guides/v2.2/cloud/cdn/cloud-fastly-custom-response.md]:
```suggestion
1.  Click **Stores** > **Settings** > **Configuration** > **Advanced** > **System**.
```

File: guides/v2.2/cloud/cdn/cloud-fastly-custom-response.md [hunk 5]
```diff
@@ -65,7 +62,7 @@ Verify that the modified source displays correctly in the browser. Then, add the
 
 1.  Click **Stores** > **Settings** > **Configuration** > **Advanced** > **System**.
 
-1.  In the right pane, expand **Full Page Cache** > **Fastly Configuration** > **Custom Synthetic Pages**.
+3.  In the right pane, expand **Full Page Cache** > **Fastly Configuration** > **Custom Synthetic Pages**.
```
Comment (dshevtsov) [inline guides/v2.2/cloud/cdn/cloud-fastly-custom-response.md]:
```suggestion
1.  In the right pane, expand **Full Page Cache** > **Fastly Configuration** > **Custom Synthetic Pages**.
```

File: guides/v2.2/cloud/cdn/cloud-fastly-custom-response.md [hunk 6]
```diff
@@ -4,9 +4,6 @@ title: Customize error pages
 redirect_from:
    - /guides/v2.2/cloud/configure/fastly-vcl-extend-timeout.html
    - /guides/v2.3/cloud/configure/fastly-vcl-extend-timeout.html
```
Comment (dobooth) [inline guides/v2.2/cloud/cdn/cloud-fastly-custom-response.md]: Do we want to redirect from 2.3 down to 2.2?

Comment (dshevtsov) [reply guides/v2.2/cloud/cdn/cloud-fastly-custom-response.md]: I assume that this is a symlinked topic. So, in fact, both redirected links 2.2 and 2.3 will go to 2.3.

Comment (dobooth) [reply guides/v2.2/cloud/cdn/cloud-fastly-custom-response.md]: Right. With symlinked files and multiple redirects, bottom one wins.

File: guides/v2.2/comp-mgr/extens-man/extensman-uninst-final.md [hunk 7]
```diff
@@ -1,10 +1,6 @@
 ---
 group: software-update-guide
-subgroup: 06_UseExtMan
 title: Step 4. Uninstall
```
[the remark this answers is not in the document]
Comment (dshevtsov) [reply guides/v2.2/comp-mgr/extens-man/extensman-uninst-final.md]: This is out of scope in this pull request.
I have a task to remove obsolete page parameters throughout devdocs in my back log. But feel free to create an issue or fix this in a pull request.

**Comment by dshevtsov** [comment]
running tests

**Pull request merged**